### PR TITLE
bugfix/FOUR-15393: Welcome Screen Filtering Project Assets is Not Very Automatic

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
List is not updated automatically, we need to click on an arrow for filtering on the table to update it.
## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-15393

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fb2c715d53.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fb2c715d53.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fb2c715d53.engk8s.processmaker.net

File: resources/js/processes/designer/RecentAssets.vue [hunk 1]
```diff
@@ -129,10 +130,19 @@ export default {
...
+    waitingForUpdate() {
+      setTimeout(() => {
+        this.performSearch();
+      }, 500);
+    },
     /**
      * This method sends users's input criteria to filter specific tasks or requests
      */
     performSearch() {
+      console.log("jiji");
```
Comment (fagubla) [inline resources/js/processes/designer/RecentAssets.vue:145]: remove console

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/processes/designer/RecentAssets.vue [hunk 2]
```diff
@@ -129,10 +130,19 @@ export default {
       this.showInput = !this.showInput;
       this.hideDropdown = false;
     },
+    /**
+     * This function waits for the selectedTypes array to be updated
+     */
+    waitingForUpdate() {
```
Comment (fagubla) [inline resources/js/processes/designer/RecentAssets.vue:136]: this is not working as the expected

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fb2c715d53.engk8s.processmaker.net

**Pull request merged**